COPYING.md

```diff
@@ -288,33 +242,6 @@
          See the License for the specific language governing permissions and
          limitations under the License.
 
-## lionshead
-
-    Code has been taken from https://github.com/natefoo/lionshead and loosely
-    translated to C++ laced with Qt.
-
-    MIT License
-
-    Copyright (c) 2017 Nate Coraor
-
-    Permission is hereby granted, free of charge, to any person obtaining a copy
-    of this software and associated documentation files (the "Software"), to deal
-    in the Software without restriction, including without limitation the rights
-    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-    copies of the Software, and to permit persons to whom the Software is
-    furnished to do so, subject to the following conditions:
-
-    The above copyright notice and this permission notice shall be included in all
-    copies or substantial portions of the Software.
-
-    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-    SOFTWARE.
-
```

Comment:
this was added in 82208be49eecc07f9c6a38365876c28273529192, that made changes to `systeminfo`, which we still use, so I think this stays

Reply:
ah nvm. Hadn't found anything about it on the git history, I'll add that back when I come back home :p